package.json

```diff
@@ -24,6 +24,8 @@
     "dev": "lerna run --parallel --scope @firebase/* --scope firebase --scope rxfire dev",
     "build": "lerna run --scope @firebase/* --scope firebase --scope rxfire prepare",
```

Comment:
replace `prepare` with `build`, so it doesn't run swapping public typings in local builds.

Reply:
It won't (see comment above) but it is better to run the expected command. Changed.